src/cart/crt_init.c

```diff
@@ -292,8 +292,7 @@ crt_init_opt(crt_group_id_t grpid, uint32_t flags, crt_init_options_t *opt)
 	if (gdata_init_flag == 0) {
 		rc = data_init(server, opt);
 		if (rc != 0) {
-			D_ERROR("data_init failed, rc(%d) - %s.\n",
-				rc, strerror(rc));
+			D_ERROR("data_init failed, "DF_RC"\n", DP_RC(rc));
 			D_GOTO(out, rc = -rc);
```

Comment:
This line is wrong as well, it changes the sign of rc but it shouldn't do.
```suggestion
			D_GOTO(out, rc);
```
I guess at some point data_init() was changed from returning a system to DAOS error code.